branch_project

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -xe
```

Comment:
Note to self: I should update this to just `-e`. `-xe` is the debug version.

Having said that, @ehelms raised that it can be difficult to figure out what's going on. Would it make sense to have a `DEBUG` env var that calls `set -x`?